src/lib.rs

```diff
@@ -16,14 +16,17 @@ const INS_GET_VERSION: u8 = 6; // Instruction code to get app version from the L
 const INS_SIGN_TRANSACTION: u8 = 2; // Instruction code to sign a transaction on the Ledger
 const NETWORK_ID: u8 = 'W' as u8; // Instruction parameter 2
 const RETURN_CODE_OK: u16 = 36864; // APDUAnswer.retcode which means success from Ledger
-const CHUNK_SIZE: usize = 128; // Chunk size to be sent to Ledger
+const CHUNK_SIZE: usize = 250; // Chunk size to be sent to Ledger
```

Comment:
I don't recall was the size of this chunk 128 because of historical reason or due to the capabilities of the Nano S. Can you confirm Nano S works with this increased?

Reply:
yes. this is specifically to handle large txs. see `Case 2` in https://github.com/dj8yfo/app-near-rs README

Reply:
additionally reconfirmed that this chunk size is fine with `app-near` 1.2.1 for nanos, downloaded from Ledger Live